hw/ip/i2c/data/i2c.hjson

```diff
@@ -107,6 +109,8 @@
       hwaccess: "hrw"
       hwext: "true"
       hwre: "true"
+      tags: [// ro registers - exclude init and write-read check
```

Comment:
Make the comment more meaningful, like: "Reading uninitialized RDATA results in X-propagation; exclude from all checks."
Likewise for others.